CONTRIBUTING.md

```diff
@@ -125,79 +185,84 @@ request.
+1. Builds the MicroProfile language server and places the jar in 
 `vscode-quarkus/server/`.
-2. Builds the Quarkus jdt.ls extension and places the jar in 
+2. Builds the MicroProfile jdt.ls extension and places the jar in 
 `vscode-quarkus/jars/`.  
 
 In addition to `npm run build`, there are two more build scripts:  
 `npm run build-server` only builds the Quarkus language server and places the jar in `vscode-quarkus/server/`.  
 `npm run build-ext` only builds the Quarkus jdt.ls extension and places the jar in `vscode-quarkus/jars/`.
 
 ### Running vscode-quarkus
-Step 1. Open `vscode-quarkus/` in VSCode.  
+**Step 1.** Open `vscode-quarkus/` in VSCode.  
 
-Step 2. Open the Debugging tab, select and run 
+**Step 2.** Open the Debugging tab, select and run 
 "Launch Extension (vscode-quarkus)" at the top left.
 ![](images/runExtension.png)
 
 ## Debugging  
-### Debugging the Quarkus language server:
+### Debugging the MicroProfile language server:
 In an IDE of your choice, set the debugger configuration to connect
-to localhost, port 1064.  
+to localhost, port 1064.
 
-If using VSCode, open `quarkus-ls/quarkus.ls/` in VSCode. The proper
+If using VSCode, open `quarkus-ls/microprofile.ls/` in VSCode. The proper
 debugger configurations are already defined in `.vscode/`.
-There should be a "Debug (Attach) - Remote (quarkus.ls)" option
+There should be a "Debug (Attach) - Remote (microprofile.ls)" option
 at the top left of the Debugging tab.
 ![](images/runDebugger.png)  
 
-The JVM arguments used to start the Quarkus language
+The JVM arguments used to start the MicroProfile language
 server are specified
 [here](https://github.com/redhat-developer/vscode-xml/blob/35c122edcce09038e853dfab112dd76813302034/src/javaServerStarter.ts#L26).
 
-### Debugging the Quarkus jdt.ls extension:
-Only Eclipse can be used to debug the Quarkus jdt.ls extension.  
+### Debugging the MicroProfile jdt.ls extension:
+Only Eclipse can be used to debug the MicroProfile jdt.ls extension.  
 
-Step 1. Open the jdt.ls source code in a new workspace in Eclipse by
+**Step 1.** Open the jdt.ls source code in a new workspace in Eclipse by
 following the setup
 steps in the jdt.ls GitHub repository 
 [here](https://github.com/eclipse/eclipse.jdt.ls#first-time-setup).  
 
-Step 2. In the same workspace, import the projects in
-`quarkus-ls/quarkus.jdt`.
+**Step 2.** In the same workspace, import the projects in
+`quarkus-ls/microprofile.jdt`.
 
-Step 3. In the Debug dropdown menu, open "Debug Configurations...".  
+**Step 3.** In the Debug dropdown menu, open "Debug Configurations...".  
 ![](images/debugConfigMenu.png)  
 
-Step 4. Create a new "Remote Java Application" launch configuration.
-Set the following settings and click "Apply".  
+**Step 4.** Create a new "Remote Java Application" launch configuration.  
+Set the following settings and click "Apply":  
+```
+Project: com.redhat.microprofile.jdt.quarkus
```

Comment:
It should be better to speak about com.redhat.microprofile.jdt.core (because it's not linked to Quarkus, just Microprofile), please update that and the linked image.

Reply:
fixed